### ISSUE TITLE
pf and pt dcline active power with same sign in psse parser

### ISSUE THREAD

**Issue by iagochavarry** [lanl-ansi/PowerModels.jl#794]
Shouldn't the active power in each of the dcline terminations have opposite signs (one end is absorbing - _positive sign_ - and the other is injecting - _negative sign_)?

I think in matpower convention `pt` is negative and `pf` is positive.

In psse parser, `power_demand` have the same sign.

```julia
        power_demand = dcline["MDC"] == 1 ? abs(dcline["SETVL"]) : dcline["MDC"] == 2 ? abs(dcline["SETVL"] / pop!(dcline, "VSCHD") / 1000) : 0
        ...
        sub_data["pf"] = power_demand
        sub_data["pt"] = power_demand
```

In matpower parser, the `pt` is set to be negative 

```julia
        dcline["pt"] = -dcline["pt"] 
```

**Comment by ccoffrin** [comment]
@jbarberia you do know the PSSE convention on this point?

**Comment by jbarberia** [comment]
@ccoffrin sorry for the delay in replying.

In PSSE, the desired power value is set in the rectifier (SETVL> 0) or in the inverter (SETVL <0). In both cases the power-flow goes from the rectifier to the inverter.
The losses can be calculated with the dc-line voltage (VSCHD) and the dc-line resistance (RDC)

Example: 
```
SETVL = -100 [MW]
RDC = 6.2 [Ohm]
VSC = 640 [kV]

current = SETVL / VSC = 217.4 [A] 
loss = current^2 * RDC = 0.29 [MW]

pt = 100 [MW] #rectifier-side
pf = 100.29 [MW] #inverter-side
```

For a better conversion of the model from PSSE to PM it would be better to take into account the losses.

**Comment by ccoffrin** [comment]
@jbarberia thanks for this.  Based on what you say here at least it sounds like we should account for a sign change in the `pt` and `pf` values when reading/writing data files or omit these values.  Do you have a preference one way or another?

**Comment by jbarberia** [comment]
I agree with the sign change. We could also add the losses on one side or the other depending on the sign of SETVL.
